src/controllers/api/v1/swap/index.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Request, Response, NextFunction } from 'express'
+import { Container } from 'typedi'
+import { validationResult } from 'express-validator'
+import SwapService from '../../../../services/swap'
```

Comment:
you can use `module-alias`  to define root or src dirs.
so you can do like `@/services/swap`